File: Week_7/problem_1-Winter_sale.py
# https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/A

discount_percentage, price_after_discount = map(float, input().split())

initial_price = 100
actual_pricing_percentage = initial_price - discount_percentage

price_without_discount = (price_after_discount * initial_price) / actual_pricing_percentage

print(f"{price_without_discount:.2f}")

File: Week_7/problem_1-Winter_sale.c
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/A

#include <stdio.h>
#include <limits.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

int main()
{
    float discount_percentage;
    int price_after_discount;
    scanf("%f %d", &discount_percentage, &price_after_discount);

    int initial_price = 100;
    float actual_pricing_percentage = initial_price - discount_percentage;

    float price_without_discount = (price_after_discount * initial_price) / actual_pricing_percentage;

    printf("%0.2f", price_without_discount);

    return 0;
}
